@here/harp-datasource-protocol/tsconfig.json

```diff
@@ -2,13 +2,27 @@
     "compilerOptions": {
         "target": "es2017",
         "module": "commonjs",
+        "outDir": "dist/cjs",
         "sourceMap": true,
         "declaration": true,
+        "declarationMap": true,
+        "composite": true,
+        "strict": true,
         "strictNullChecks": true,
         "noImplicitAny": true,
         "noImplicitReturns": true,
         "noImplicitThis": true,
-        "noUnusedLocals": false,
-        "downlevelIteration": true
-    }
+        "noUnusedLocals": false
```

Comment:
I think I will add`tsconfig.common.json` in the root directory of harp.gl and included it in the the `@here/harp-*` packages